ptsprojects/mynewt/post_tc.py

```diff
@@ -23,7 +23,7 @@
 
 CONFIG_PROC = None
 # XXX: Fill me - nrfjprog path example: /home/user/tool/nrfjprog
-CONFIG_PATH = None
+CONFIG_PATH = '/usr/bin/nrfjprog'
```

Comment:
I guess it's not in the scope of this PR

Reply:
Sure, it isn't :)